packages/rancher-logging/package.yaml

```diff
@@ -1,5 +1,5 @@
 url: https://kubernetes-charts.banzaicloud.com/charts/logging-operator-3.17.10.tgz
-version: 101.1.0
+version: 102.0.0
```

Comment:
102 would be for a Rancher 2.8 version line, I think this should be 101.2.0 if the change merits a minor version bump

Reply:
Discussed offline, lgtm